index.html

```diff
@@ -84,14 +84,159 @@ <h1>Hey there.<span class="span__br"><br /></span> I’m Mateo.
             </ul>
           </div>
 
-      <div class="arrow_button">
-        <a href="#">
-          <img src="./assets/button/arrowbutton.svg">
-        </a>
+        <div class="arrow_button">
+          <a href="#">
+            <img src="./assets/button/arrowbutton.svg">
+          </a>
+        </div>
       </div>
     </div>
+  </section>
+
+  <!----Projects Section---------->
+
+  <section id="projects">
+    <div class="projects">
+      <h2 class="project_title">
+        My Recent <span class="span__br"><br /></span>Works
+      </h2>
+      <div class="projects_h2_icon"></div>
+      <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
```

Comment:
- [ ] [Optional] Great job on using the `<article>` tag for the card! 👍 However, according to the design, the main `<article>` tag that wraps the card container should have a flex layout applied to it. This will help align the content of each project using flexbox. Currently, the flex layout is missing. Make sure to apply the flexbox properties to the main container of the card to match the design requirements. Keep up the good work! 💪